build/check-style.sh

```diff
@@ -95,7 +95,7 @@ TestImportNames() {
 }
 
 TestIneffassign() {
-  ! ineffassign . | grep -vF 'ineffectual assignment to sqlDollar' | grep -vF '.pb.go' # https://github.com/gogo/protobuf/issues/149
+  ! (ineffassign . || true) | grep -vF 'ineffectual assignment to sqlDollar' | grep -vF '.pb.go' # https://github.com/gogo/protobuf/issues/149
```

Comment:
The `!` binds weaker than the others. I parenthesized throughout to make this clearer.